src/main/java/lotto/model/LottoGame.java

```diff
@@ -17,8 +17,9 @@ public LottoGame(LottoTickets lottoTickets, WinningTicket winningTicket) {
         this.winningTicket = winningTicket;
     }
 
-    public RankResults start() {
-        Ranks ranks = new Ranks(lottoTickets.getRanks(winningTicket));
+    public RankResults start(LottoNumber bonusNumber) {
+        winningTicket.validateBonusNumber(bonusNumber);
+        Ranks ranks = new Ranks(lottoTickets.getRanks(winningTicket, bonusNumber));
         List<RankResult> rankResults = ranks.groupBy();
         return new RankResults(rankResults);
```

Comment:
```suggestion
        return new RankResults(ranks.groupBy());
```

현재 임시변수의 값을 사용하는 곳이 없어서 바로 넘겨도 될것 같습니다. 😄